es6-lib/util/metrics.js

```diff
@@ -9,12 +9,59 @@ var conf = config();
 
 class Metrics {
   constructor() {
-    setInterval(this._sampleMemory.bind(this), 1000);
+    this._resetState();
     this._gc = gcstats();
     this._gc.on('stats', _.throttle(this._onGcStat, 1000).bind(this));
   }
 
-  static heapdump(req, res) {
+  _resetState() {
+    this._state = {
+      http: {
+        status: {}
+      },
+      gc: {},
+      memory: {}
+    };
+  }
+
+  metrics(req, res) {
+    res.status(200).send(JSON.stringify(this.state));
+    this._resetState();
```

Comment:
Are the metrics contained here instantaneous samples or diffs from the last time metrics were collected? Basically, will the metrics returned be affected by the time elapsed between calls?